examples/diagnostics_and_criticism/model_averaging.ipynb

```diff
@@ -2,14 +2,8 @@
  "cells": [
```

Comment:
<p>and here:</p><p> </p><pre class="ql-syntax" spellcheck="false">ppc_2 = pm.sample_posterior_predictive(trace_2, model=model_2, progressbar=False)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>